### PR TITLE
Update profile picture for user

### PR DESCRIPTION
**Pull request by imarban**

### CODE REVIEW

File: src/test/groovy/vincool/CloudinaryServiceSpec.groovy [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+    def setup() {
+    }
+
+    def cleanup() {
+    }
+
+    void "test something"() {
+        expect:"fix me"
+            true == false
```
Comment (luisalduucin) [inline src/test/groovy/vincool/CloudinaryServiceSpec.groovy]: Can you change it to pass the test ?

**Review by luisalduucin — CHANGES_REQUESTED** [review]

File: src/test/groovy/vincool/CloudinaryServiceSpec.groovy (hunk 1 above)
Comment (imarban) [reply src/test/groovy/vincool/CloudinaryServiceSpec.groovy]: Done

**Review by luisalduucin — APPROVED** [review]

File: grails-app/controllers/vincool/ProfileController.groovy [hunk 2]
```diff
@@ -30,6 +32,19 @@ class ProfileController {
 
         def user = springSecurityService.getCurrentUser()
 
+        def profilePicture = request.getPart("picture")
+
+        if (profilePicture) {
+            def fileName = org.apache.commons.io.FilenameUtils.getName(user.profilePictureUrl)
```
Comment (blzb) [inline grails-app/controllers/vincool/ProfileController.groovy]: Why is this not an import ?

File: grails-app/services/vincool/cloudinary/CloudinaryUploadResult.groovy [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    String resourceType
+    Date createdAt
+    long bytes
+    String type
+    String url
+    String secureUrl
+
+
+    @Override
+    public String toString() {
```
Comment (blzb) [inline grails-app/services/vincool/cloudinary/CloudinaryUploadResult.groovy]: If this is a required way of to string it's ok. If this is just debugging maybe it should be better to use @ToString http://mrhaki.blogspot.mx/2011/04/groovy-goodness-easy-tostring-creation.html

**Pull request merged**